docs/sources/setup/_index.md

```diff
@@ -16,13 +16,15 @@ aliases:
 
 There are different options to set up and run Beyla:
 
-1. [As a standalone Linux process]({{< relref "./standalone.md" >}}).
-2. [With Docker to instrument a process running in a container]({{< relref "./docker.md" >}}).
-3. [As a Kubernetes DaemonSet using Helm]({{< relref "./kubernetes-helm.md" >}})
-4. [As a Kubernetes DaemonSet or as a sidecar container (manual process)]({{< relref "./kubernetes.md" >}})
+1. [With Docker to instrument a process running in a container]({{< relref "./docker.md" >}}).
+1. [As a Kubernetes DaemonSet using Helm]({{< relref "./kubernetes-helm.md" >}})
+1. [As a Kubernetes DaemonSet the Grafana Alloy Helm chart]({{< relref "./helm-alloy.md" >}})
+1. [As a Kubernetes DaemonSet or as a sidecar container (manual process)]({{< relref "./kubernetes.md" >}})
 
 For information on configuration options and data export modes, see the [Configure Beyla]({{< relref "../configure/_index.md" >}}) documentation.
 
+You can run Beyla as a standalone process. This approach is suitable for local development or advanced use cases. Consult the documentation in the [git repository](https://github.com/grafana/beyla/blob/main/docs/sources/setup/standalone.md) to learn how to set up Beyla as a standalone process.
```

Comment:
I think it's better to change this statement,

From:
`This approach is suitable for local development or advanced use cases`
To:
`This approach is suitable for running with bare metal installations, in virtual machines, for local development or advanced use cases`